src/main/java/by/onliner/page/pages/ShoppingCartPage.java

```diff
@@ -1,26 +1,28 @@
-package by.onliner.page;
+package by.onliner.page.pages;
 
-import by.onliner.util.MoveTo;
+import by.onliner.driver.DriverManager;
+import by.onliner.page.AbstractPage;
+import by.onliner.util.ActionsHelper;
 import by.onliner.util.Waiter;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
 public class ShoppingCartPage extends AbstractPage {
 
     @FindBy(xpath = "//div[contains(@class, 'cart-form__title')]")
-    WebElement openedShoppingCartMessage;
+    private WebElement openedShoppingCartMessage;
 
     @FindBy(className = "cart-form__control")
-    WebElement removeFromCartButton;
+    private WebElement removeFromCartButton;
 
     @FindBy(xpath = "//div[contains(@class, 'cart-form__offers-unit_primary')]")
-    WebElement Item;
+    private WebElement cartUnit;
```

Comment:
update variable name. Element name should contain element type, ex. submitButton, errorLabel and etc. 
Please make sure all your variables follow the same pattern

Reply:
Renamed to cartUnitForm